lib/bitstyles_phoenix/component/form.ex

```diff
@@ -100,7 +106,7 @@ defmodule BitstylesPhoenix.Component.Form do
             *
           </span>
         </label>
-        <input id="user_name" maxlength="255" name="user[name]" required="required" type="text"/>
+        <input id="user_name" name="user[name]" type="text" maxlength="255" required="required"/>
```

Comment:
In all the existing test cases in this module that have a git diff, the diff is only about reordering attributes.